MapboxVision/src/main/java/com/mapbox/vision/video/videosource/camera/Camera2VideoSourceImpl.kt

```diff
@@ -167,7 +170,14 @@ class Camera2VideoSourceImpl(
                         }
 
                         override fun onConfigureFailed(cameraCaptureSession: CameraCaptureSession) {
-                            throw IllegalStateException("Failed to configure Camera ${cameraCaptureSession.device.id}!")
+                            if (attemptNumber <= attemptMax) {
```

Comment:
will _attemptMax_ be converted to _Long_ each time?

Reply:
ptal after refactoring